app/src/Version.h.in

```diff
@@ -1,8 +1,9 @@
 #define VERSION_MAJOR @APP_VERSION_MAJOR@
 #define VERSION_MINOR @APP_VERSION_MINOR@
 #define VERSION_MAINTENANCE @APP_VERSION_MAINTENANCE@
-#define BUILD_ID "@APP_BUILD_ID@"
-#define BUILD_CHANNEL "@APP_BUILD_CHANNEL@"
+
 #define BUILD_TYPE "@APP_BUILD_TYPE@"
+#define BUILD_ID_STR "@GIT_DESCRIBE@ [@APP_BUILD_TYPE@]"
```

Comment:
Please get rid of the square brackets. Now that I saw them in the welcome frame, I think they look out of place.